sequencer/src/lib.rs

```diff
@@ -762,6 +767,10 @@ pub mod testing {
             self.builder_port
         }
 
+        pub fn url(&self) -> Url {
+            self.url.clone()
+        }
```

Comment:
This should probably be called `l1_url` as that's what it really represents. I'm not sure why it had such a generic name as `url` to begin with, as this is a struct representing an entire network and just `url` is really vague